doc/prop.xml

```diff
@@ -1307,14 +1307,27 @@ true
     (<Ref Prop="IsLatticeDigraph"/>) which is distributive. That is to say,
     the functions <Ref Oper="PartialOrderDigraphMeetOfVertices"/> and
     <Ref Oper="PartialOrderDigraphJoinOfVertices"/> distribute over each other.<P/>
-    
+
     Equivalently, a <E>distributive lattice digraph</E> is a <E>lattice digraph</E>
     in which the <E>lattice digraphs</E> representing <M>M3</M> and <M>N5</M> are
     not embeddable as lattices
     (see <URL>https://en.wikipedia.org/wiki/Distributive_lattice</URL> and
-    <Ref Prop="IsLatticeEmbedding" 
+    <Ref Oper="IsLatticeEmbedding"
     Label="for digraphs and a permutation or transformation"/>).<P/>
 
+    <Alt Only="HTML">
+        <![CDATA[
+        <p>
+        <center>
+            <img height="200" src="m3.png"/>
+            &nbsp;&nbsp;&nbsp;&nbsp;
+            <img height="200" src="n5.png"/>
+            <figcaption>The lattices <e>M3</e> and <e>N5</e>.</figcaption>
+        </center>
+        </p>
```

Comment:
The HTML standard says that `figcaption` can only appear as the child of a `figure` element, so I guess this is invalid.  Should we use `<figure>` here instead of `<p><center>` ?  (It's more semantic anyway!)

Reply:
Great point! I've updated this to use figure, plus center, and the output looks good I think:
![Screenshot 2024-04-17 at 09 12 59](https://github.com/digraphs/Digraphs/assets/6686071/f3db3208-5ea6-4bae-a75d-122e00c9edc0)

Reply:
Strictly this is still invalid HTML, since our `figcaption` is a (direct) child of a `center` element and not a `figure` element.  I can definitely see we want them centered though.

The "proper" solution would be to add
```css
figure {
    text-align: center;
}
```
to `manual.css`, but that's a matter for GAPDoc.

For now, your solution seems the best one.

Reply:
One last thing though: do we want a `p` element inside `figcaption`?  I think it's standard just to put text directly into `figcaption`, unless there's some reason that's not working.

Reply:
Thanks @mtorpey. For the first comment, if I center then figure, this will resolve that issue, right?  Or I guess we can adjust the css in doc/ to include your proper solution? 

About the <p> in <figcaption> I inserted that because o/w the caption was very close to the figures, and with it, there is a reasonable vertical space. If there's a proper solution for that I'll be happy to learn it!

Reply:
For both of these, the proper fix is definitely with css.  The figcaption spacing issue would be solved with
```css
figcaption {
    margin: 1em 0;
}
```
but from what I can tell GAPDoc produces the css file when we compile the doc – unless I'm missing something, we can't specify our own without changing GAPDoc itself.

If we can't change the CSS ourselves, then I like your `<center><figure>` solution, and `<figcaption><p>` seems a reasonable workaround – I believe it's at least legal.

Reply:
Good point! It seems that we can add our own style sheets, from the GAPDoc manual:

> The generated HTML files refer to the file manual.css which conforms to the W3C specification CSS 2.0, see https://www.w3.org/TR/REC-CSS2, and the javascript file manual.js (only in browsers which support CSS or javascript, respectively; but the HTML files are also readable without any of them). To add a style mystyle one or both of mystyle.css and mystyle.js must be provided; these can overwrite default settings and add new javascript functions. For more details see the comments in manual.js.

It's not completely clear to me how we do this, any thoughts @mtorpey ?

Reply:
I've just been reading this myself.  It looks like we can maybe add another file called `mystyle.css` and the Javascript will import it automatically.  Might be a bit complicated though – maybe we should merge this as-is and make a separate issue?